kiwi/container/docker.py

```diff
@@ -113,12 +116,24 @@ def create(self, filename):
             [container_dir, self.container_tag]
         )
 
-        Command.run(
-            ['umoci', 'init', '--layout', container_dir]
-        )
-        Command.run(
-            ['umoci', 'new', '--image', container_name]
-        )
+        if base_image:
+            uncompressor = Compress(base_image)
+            uncompressor.uncompress(True)
+            self.uncompressed_image = uncompressor.uncompressed_filename
+
+            Command.run([
+                'skopeo', 'copy',
+                'docker-archive:{0}'.format(self.uncompressed_image),
+                'oci:{0}'.format(container_name)
+            ])
```

Comment:
This is done to make sure we keep all the layers. We need to make a new layer on top of an existing image, so in that case the `umoci init --layout` and `umoci new --image` are not required. The sequence is `umoci unpack` the existing one, synchronize the modified rootfs, and finally  `umoci repack`. The `RootImport` unpacked that image before, but it doesn't exist anymore at the time, as it is unpacked to a temp folder.

Then I decided to keep the original image inside the /image because I wanted to make sure that the same image file is used in both cases. Imagine we run `system prepare` and `system create` as separate steps and the original image gets modified in between, because is remote, because the user deleted it... I wanted to leave an self-sufficient structure after `system prepare`